src/test/java/core/basesyntax/service/impl/FileWriterServiceImplTest.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileWriterServiceImplTest {
+    private static final String RESOURCES_PATH_STRING = "src/test/resources/";
+    private static final Path INPUT_PATH = Path.of(RESOURCES_PATH_STRING + "input.csv");
+    private static final Path UNEXISTING_PATH = Path.of(RESOURCES_PATH_STRING + "output.csv");
+    private static FileWriterService writer;
+    private static String testData;
+
+    @BeforeClass
+    public static void beforeAll() {
+        testData = "line1Header\n"
+                + "line2Content\n"
+                + "line3Content";
+    }
+
+    @Before
+    public void beforeEach() {
+        writer = new FileWriterServiceImpl(INPUT_PATH);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void write_withNullPath_notOk() {
```

Comment:
`<methodUnderTest>_<state>_<expectedBehavior>`
writeToFile(), not write()

Reply:
+Corrected.